cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/security/trustedproxy/TrustedProxyAuthorizationService.java

```diff
@@ -20,12 +21,18 @@
  */
 public class TrustedProxyAuthorizationService extends AbstractLifeCycle implements AuthorizationService {
 
+  private static final Credential NO_CREDENTIAL = new Credential() {
+    @Override
+    public boolean check(Object credentials) {
+      return false;
+    }
+  };
   private final UserStore _adminUserStore;
   private final Pattern _trustedProxyIpPattern;
 
   TrustedProxyAuthorizationService(List<String> userNames, String trustedProxyIpPattern) {
     _adminUserStore = new UserStore();
-    userNames.forEach(u -> _adminUserStore.addUser(u, null, new String[] { DefaultRoleSecurityProvider.ADMIN }));
+    userNames.forEach(u -> _adminUserStore.addUser(u, NO_CREDENTIAL, new String[] { DefaultRoleSecurityProvider.ADMIN }));
```

Comment:
I am curious why this change is needed -- doesn't it introduce a behavior change?
 -- Also, in case the change is needed, can we move it to a publicly accessible place and let the relevant unit tests (e.g. `JwtAuthenticatorTest`, `JwtLoginServiceTest`, `TrustedProxyLoginServiceTest`, and `SpnegoUserStoreAuthorizationServiceTest`) access this -- i.e. instead of defining their own static vars in unit tests?

Reply:
right. the change is needed as `addUser` is now requiring non-null credentials. A NPE would throw if null is passed (regardless if it's being used or not).

Reply:
agree on reusing it - what would be a good place to define it ?

Reply:
Thanks for the clarification! 
> agree on reusing it - what would be a good place to define it ?

Would a utility class work?
```
/*
 * Copyright 2020 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
 */

package com.linkedin.kafka.cruisecontrol.servlet.security;

import org.eclipse.jetty.util.security.Credential;


public final class SecurityUtils {

  private SecurityUtils() {
  }

  public static final Credential NO_CREDENTIAL = new Credential() {
    @Override
    public boolean check(Object credentials) {
      return false;
    }
  };
}
```